src/Orchard.Web/Modules/Orchard.SecureSocketsLayer/Handlers/SslSettingsPartHandler.cs

```diff
@@ -22,5 +34,10 @@ public class SslSettingsPartHandler : ContentHandler {
                 Position = "2"
             });
         }
+
+        private void Invalidate(SslSettingsPart content) {
+            _signals.Trigger($"SslSettingsPart_{content.Id}");
```

Comment:
I will keep it but I think it's not necessary. And I don't see it used anywhere else either.